tetrateci/create_istio_release.sh

```diff
@@ -44,13 +44,8 @@ if [[ ${BUILD} == "fips" ]]; then
 fi
 
 # HACK : default manifest from release builder is modified
-echo "Generating the docker manifest"
-envsubst < ${BASEDIR}/tetrateci/manifest.yaml.in > ${BASEDIR}/../release-builder/manifest.docker.yaml
-echo "  - docker" >> ${BASEDIR}/../release-builder/manifest.docker.yaml
-echo "Generating the archive manifest"
-envsubst < ${BASEDIR}/tetrateci/manifest.yaml.in > ${BASEDIR}/../release-builder/manifest.archive.yaml
-echo "  - archive" >> ${BASEDIR}/../release-builder/manifest.archive.yaml
-
+echo "Generating the manifests"
```

Comment:
Can you add a comment here that it creates two yaml one for docker and the other for archive builds.

Reply:
Sure